development/image-syncer/external-images.yaml

```diff
@@ -1,6 +1,7 @@
 targetRepoPrefix:  "eu.gcr.io/kyma-project/external/"
 images:
 - source: "alpine:3.12.0"
+- source: "alpine:3.12.3"
```

Comment:
Why do not refer the alpine:3.12 instead. It points to the latest micro version of 3.12.X

Reply:
I think we want to have only fixed versions of the images in our GCR. Correct me if I'm wrong :D